### ISSUE TITLE
SBT assembly is not building the standalone jar 

### ISSUE THREAD

**Issue by jumpnrun88** [YotpoLtd/metorikku#204]
Hello 

When I am trying to build the standalone jar with sbt assembly , it is not creating the jar . It is geting halted after the test is completed 

but sbt package is somehow working .

19/07/02 16:29:52 WARN CacheManager: Asked to cache already cached data.
[info] - Test Metorikku should not fail on invalid query when ignoreOnFailures is set to true
[info] ScalaTest
[info] Run completed in 1 minute, 12 seconds.
[info] Total number of tests run: 6
[info] Suites: completed 1, aborted 0
[info] Tests: succeeded 6, failed 0, canceled 0, ignored 0, pending 0
[info] All tests passed.
[info] Passed: Total 6, Failed 0, Errors 0, Passed 6
[success] Total time: 299 s, completed 2/07/2019 04:33:34 PM
[IJ]>

**Comment by lyogev** [comment]
I don't think it halts, it just takes a very long time depending on your computer.
Try with some memory configurations:
```
-Xms2048M
-Xmx2048M
-Xss6M
-XX:MaxPermSize=512M
-XX:ReservedCodeCacheSize=256M
```
